Engine/source/gfx/gl/sdl/gfxGLDevice.sdl.cpp

```diff
@@ -58,7 +58,16 @@ void EnumerateVideoModes(Vector<GFXVideoMode>& outModes)
       GFXVideoMode outMode;
       outMode.resolution.set( mode.w, mode.h );
       outMode.refreshRate = mode.refresh_rate;
-      outMode.bitDepth = SDL_BYTESPERPIXEL( mode.format );
+
+      // BBP = 32 for some reason the engine knows it should be 32, but then we
+      // add some extra code to break what the engine knows.
+      //outMode.bitDepth = SDL_BYTESPERPIXEL( mode.format );     // sets bitdepths to 4
+      //outMode.bitDepth = SDL_BITSPERPIXEL(mode.format);        // sets bitdepth to 24
+      
+      // hardcoded magic numbers ftw
+      // This value is hardcoded in DX, probably to avoid the shenanigans going on here
+      outMode.bitDepth = 32;                                   
```

Comment:
I'd prefer if we figured out what's going wrong :)

Reply:
interesting that the note is that DirectX has it hardcoded as 32bits...I wonder why SDL is incorrectly detecting the bitdepth. We'd PROBABLY be okay with hardcoding it as...any computer since 2000 I think would support 32bit color depth, but it would be interesting to see why SDL is causing issues.

Reply:
Fwiw, did try and do a trace on that at the time of submission, and ended up very much lost in the weeds. end of the day, it's a step towards consistent behavior across platforms a least